modules/opensearch/opensearch.module

```diff
@@ -256,7 +256,7 @@ function opensearch_admin_settings($form_state) {
   $form['opensearch']['opensearch_url'] = array(
     '#type' => 'textfield',
     '#title' => t('Search service URL'),
-    '#description' => t('URL to the Ting search webservice, e.g. http://opensearch.addi.dk/3.0/'),
+    '#description' => t('URL to the Ting search webservice, e.g. https://opensearch.addi.dk/b3.5_5.0/),
```

Comment:
There is a syntax error here. A trailing `'` is missing.

This is revealed by the CI output:

```
WD php: ParseError: syntax error, unexpected ''#default_value''          [error]
(T_CONSTANT_ENCAPSED_STRING), expecting ',' or ')' in drupal_load()
(line 261 of
/home/ubuntu/build/profiles/ding2/modules/opensearch/opensearch.module).
WD php: Warning: Cannot modify header information - headers already    [warning]
sent by (output started at
/home/ubuntu/.composer/vendor/drush/drush/includes/output.inc:38) in
drupal_send_headers() (line 1486 of
/home/ubuntu/build/includes/bootstrap.inc).
ParseError: syntax error, unexpected &#039;&#039;#default_value&#039;&#039; (T_CONSTANT_ENCAPSED_STRING), expecting &#039;,&#039; or &#039;)&#039; in drupal_load() (line 261 of /home/ubuntu/build/profiles/ding2/modules/opensearch/opensearch.module).
Drush command terminated abnormally due to an unrecoverable error.       [error]
```